indexing/src/main/java/com/google/enterprise/cloudsearch/sdk/indexing/StructuredData.java

```diff
@@ -354,16 +359,21 @@ private NamedProperty getProperty(String propertyName, Collection<Object> values
       if (nonNullValues.isEmpty()) {
         return null;
       }
-      if (!isRepeated) {
-        return propertyBuilder.getNamedProperty(
-            propertyName, Collections.singletonList(valueConverter.convert(nonNullValues.get(0))));
-      } else {
-        return propertyBuilder.getNamedProperty(
-            propertyName,
-            nonNullValues
-                .stream()
-                .map(v -> valueConverter.convert(v))
-                .collect(Collectors.toList()));
+      try {
+        if (!isRepeated) {
+          return propertyBuilder.getNamedProperty(
+              propertyName, Collections.singletonList(valueConverter.convert(nonNullValues.get(0))));
+        } else {
+          return propertyBuilder.getNamedProperty(
+              propertyName,
+              nonNullValues
+                  .stream()
+                  .map(v -> valueConverter.convert(v))
```

Comment:
value converter::convert while you're at it :D

Reply:
v-> valueConverter.convert(v) can also be written as valueConverter::convert

I am fine with existing pattern as well :)

Reply:
Ok, thanks for the clarification.